deploy/run_with_summon.sh

```diff
@@ -20,6 +20,10 @@ finish() {
 }
 trap finish EXIT
 
+# Will print platform regardless of GKE or Openshift
+# Will only print version if Openshift
+echo "Running tests on: ${PLATFORM} ${OPENSHIFT_VERSION}"
```

Comment:
can we use `announce` instead of echo? it will def get lost in the logs with just echo.